ChangeLog

```diff
@@ -124,9 +124,6 @@ modules are added.
   Closes #4509
   Closes PyCQA/astroid#999
 
-* New checker ``consider-using-namedtuple-or-dataclass``. Emitted when dictionary values
```

Comment:
Don't we still need to advertise this new checker (as optional that need to be added to the pylintrc to be used) ?

Reply:
I already added a note about the `CodeStyleChecker` with one of the last MRs. However I agree, it does make sense to mention the checker as well. I've moved it so it's a sub-point now.